artemis/amqp-connector/src/main/java/org/apache/activemq/artemis/integration/amqp/AMQPConnectorServiceFactory.java

```diff
@@ -72,6 +74,7 @@ public ConnectorService createConnectorService(String connectorName, Map<String,
       Map<String, Object> connectorConfig = new HashMap<>();
       connectorConfig.put(TransportConstants.HOST_PROP_NAME, configuration.get(TransportConstants.HOST_PROP_NAME));
       connectorConfig.put(TransportConstants.PORT_PROP_NAME, configuration.get(TransportConstants.PORT_PROP_NAME));
+      connectorConfig.put(TransportConstants.NETTY_CONNECT_TIMEOUT, "10000");
```

Comment:
Similar comment to before on the hardcoding of timeouts

Reply:
I'll add it to the list of allowed properties as well so that it can be overridden.